packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -31,15 +33,16 @@ import { PluginLoader } from './instrumentation/PluginLoader';
  */
 export class NodeTracerProvider extends BasicTracerProvider {
   private readonly _pluginLoader: PluginLoader;
+  private readonly _nodeTracers: Map<string, NodeTracer> = new Map();
 
   /**
    * Constructs a new Tracer instance.
    */
-  constructor(config: NodeTracerConfig = {}) {
+  constructor(config?: NodeTracerConfig) {
```

Comment:
I've removed the default set there because it would use an empty config as default config